cisco_duo/manifest.json

```
@@ -0,0 +1,37 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "8d3e0e2f-df52-4a12-a976-3ded71553a3a",
+  "app_id": "cisco-duo",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gain insights into Cisco Duo logs.",
+    "title": "cisco_duo",
+    "media": [],
+    "classifier_tags": [
+      "Category::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": false,
```

@mohittilala-crest just a heads-up that there's a new version of the `ddev` CLI tool that will populate a newly required `source_type_id` field in `assets > integration`. The new requirement will be rolling out soon so we might as well add it now.

```suggestion
      "auto_install": false,
      "source_type_id": 6576439,
```

---

Sure @nathanmadams, I'll take that into consideration moving forward.